core/src/util/tileID.h

```diff
@@ -16,6 +16,7 @@ struct TileID {
     
     TileID(int _x, int _y, int _z) : x(_x), y(_y), z(_z) {};
     TileID(const TileID& _rhs): x(_rhs.x), y(_rhs.y), z(_rhs.z) {};
+    TileID(const TileID&& _rhs): x(std::move(_rhs.x)), y(std::move(_rhs.y)), z(std::move(_rhs.z)) {};
```

Comment:
@tallytalwar thanks, I'll check it out. The best read discussing c++11 argument passing, that I found so far, was this series
http://www.codesynthesis.com/~boris/blog/2012/06/19/efficient-argument-passing-cxx11-part1/